### PR TITLE
[15.0][MIG] sale_force_whole_invoiceability: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT37312

### CODE REVIEW

File: sale_force_whole_invoiceability/tests/test_sale_force_whole.py [hunk 1]
```diff
@@ -26,34 +33,35 @@ def setUpClass(cls):
...
             "Only Sales Managers are allowed to force the lines to invoice",
-            cm.exception.name,
+            cm.exception.args[0],
         )
         # Once we grant permission everything goes ok
-        self.group_sales_manager.users |= self.user_demo
-        self.sale_order.with_user(
-            self.user_demo.id
-        ).force_lines_to_invoice_policy_order()
+        self.group_sales_manager.users |= self.env.user
```
Comment (pedrobaeza) [inline sale_force_whole_invoiceability/tests/test_sale_force_whole.py]: This part can be removed as now the rest is handled for the proper groups.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Thanks! :+1:

**Pull request merged**